### ISSUE TITLE
Image endpoint not cleaned up correctly

### ISSUE THREAD

**Issue by sporritt** [sporritt/jsPlumb#420]

**Comment by sporritt** [comment]
this test fails:

```
test("Image Endpoint remove", function() {
        var d1 = _addDiv("d1"), d2 = _addDiv("d2");
        _jsPlumb.makeSource(d1, {
            endpoint:[ "Image", { src:"atom.png" }]
        });

        _jsPlumb.makeTarget(d2, {
            endpoint:[ "Image", { src:"atom.png" }]
        });

        var c = _jsPlumb.connect({source:d1, target:d2});
        var ep = c.endpoints[0];

        ok(ep.canvas.parentNode != null, "endpoint 1 is in the DOM");

        _jsPlumb.detach(c);
        ok(ep.canvas.parentNode == null, "endpoint 1 is no longer in the DOM");
    });
```

**Comment by sporritt** [comment]
the endpoint's canvas should have no parent node, ie. it should no longer be in the DOM.

**Comment by sporritt** [comment]
fixed in 1.7.8
